### DISCUSSION TITLE
Sign transaction through API and broadcast

### DISCUSSION THREAD

**Discussion by ridetothefuture** [btcpayserver/btcpayserver#2697] [General]
I wanna sign transaction through API and Broadcast by using Hot Wallet. This API is missing in the documentation

**Comment by Kukks** [comment]
Please do not bypass the issue templates. We currently have this: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreOnChainWallets_CreateOnChainTransaction 
Please provide more information on what you need
